README.md

```diff
@@ -145,8 +145,8 @@ processes.
    approve.
 
 When updating the Protobuf files in a pull request, you will also need to update
-the generated Go files, which are hosted in this repository under
-[go/](go/). This can be done easily by running `./codegen/update_go.sh`,
+the generated Go and Python files, which are hosted in this repository under
+[go/](go/) and [py/](py/). This can be done easily by running `./codegen/update_[go/py].sh`,
```

Comment:
I think some unification can be done in all scripts (update / check) so that contributors only have one script to run and so that we don't do duplicate work in CI. However, this can wait if you don't want to address this in this PR...

Reply:
Done, now we only have `CI/check_codegen.sh` and `codegen/update.sh`